.github/workflows/ci.yml

```diff
@@ -55,6 +55,8 @@ jobs:
     if: github.ref == 'refs/heads/main'
     env:
       FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
@LuchoTurtle did you use your _own_ (`lucho-localhost`) AWS Access keys on Fly.io? 💭

Reply:
Yes, I did. It was just to get the deployed app working to check if everything was working correctly and so it was easier on you to change these values in the repo secrets.

Reply:
Ok. Good to make progress but ideally we should just open an issue and assign these kinds of `DevOps` chores to me. 😜